cost-analyzer/values.yaml

```diff
@@ -588,8 +588,8 @@ networkCosts:
       # In Zone contains a list of address/range that will be
       # classified as in zone.
       in-zone:
-        # Loopback
-        - "127.0.0.1"
+        # Loopback Addresses in "IANA IPv4 Special-Purpose Address Registry"
```

Comment:
To confirm, this would've incorrectly excluded this subset of loopback and would've appeared to have charged us?

Reply:
What happens on line 608? What classification do we have now for cross zone?

Reply:
> To confirm, this would've incorrectly excluded this subset of loopback and would've appeared to have charged us?

Correct. It would have been excluding these addresses: `127.0.0.0` and `127.0.0.2` to `127.255.255.255`.